src/DetailsView/components/left-nav/nav-link-handler.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { autobind } from '@uifabric/utilities';
-
+import * as React from 'react';
```

Comment:
Do we need `React` for a `ts` file?

Reply:
done... we should have a closer look at this though